akrivis-database-model/src/main/java/org/akrivis/dbmodel/ResultRepository.java

```diff
@@ -0,0 +1,30 @@
+package org.akrivis.dbmodel;
+
+import io.quarkus.hibernate.orm.panache.PanacheRepository;
+import jakarta.enterprise.context.ApplicationScoped;
+import java.util.List;
+
+@ApplicationScoped
+public class ResultRepository implements PanacheRepository<RunResult> {
+
+    public RunResult latest(long cardId) {
+        return getEntityManager()
+                .createQuery(
+                        "select r from RunResult r where card.id = :cardId order by r.runTime desc",
+                        RunResult.class)
+                .setParameter("cardId", cardId)
+                .setMaxResults(1)
```

Comment:
use HQL `max` instead to avoid fetching the whole table

Reply:
Thank you, I will change and read about that.